File: app/src/main/java/com/example/epigram/EpigramApplication.java
package com.example.epigram;

import android.app.Application;
import net.danlew.android.joda.JodaTimeAndroid;

public class EpigramApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        JodaTimeAndroid.init(this);
    }
}

File: app/src/main/java/com/example/epigram/EpigramApplication.kt
package com.example.epigram

import android.app.Application
import com.example.epigram.arch.AppModule
import net.danlew.android.joda.JodaTimeAndroid

class EpigramApplication : Application() {
    override fun onCreate() {
        AppModule.application = this
        super.onCreate()
        JodaTimeAndroid.init(this)
    }
}